webpack/assets/javascripts/react_app/API.jsx

```diff
@@ -0,0 +1,11 @@
+import ServerActions from "./actions/ServerActions"
```

Comment:
All 'jsx' files should be changed to 'js' files.  That seems to be where the consensus is going.
There isn't an special handling for 'jsx' files.